selenium_tests/course_admin/course_admin_base_test_case.py

```diff
@@ -0,0 +1,31 @@
+from urlparse import urljoin
+
+from django.conf import settings
+
+from selenium_common.base_test_case import BaseSeleniumTestCase
+from selenium_common.pin.page_objects.pin_login_page_object import \
+    PinLoginPageObject
+from selenium_tests.course_admin.page_objects\
+    .course_admin_dashboard_page_object import CourseAdminDashboardPage
+
+
+class CourseAdminBaseTestCase(BaseSeleniumTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(CourseAdminBaseTestCase, cls).setUpClass()
+
+        cls.USERNAME = settings.SELENIUM_CONFIG['selenium_username']
+        cls.PASSWORD = settings.SELENIUM_CONFIG['selenium_password']
+        cls.CANVAS_BASE_URL = settings.SELENIUM_CONFIG['canvas_base_url']
+        cls.TOOL_RELATIVE_URL = settings.SELENIUM_CONFIG['manage_course'][
+            'relative_url']
+        cls.TOOL_URL = urljoin(cls.CANVAS_BASE_URL, cls.TOOL_RELATIVE_URL)
+        cls.course_admin_dashboard_page = CourseAdminDashboardPage(cls.driver)
+        cls.course_admin_dashboard_page.get(cls.TOOL_URL)
+
+        login_page = PinLoginPageObject(cls.driver)
+        if login_page.is_loaded():
+            login_page.login_xid(cls.USERNAME, cls.PASSWORD)
```

Comment:
Yup, we've always been using an XID to login (for other projects as well), so I haven't changed the logic here.  We are using a mix of test data to test the Manage People tool, and that's reflected in the test data spreadsheet.  Let me know if there are new concerns here.